README.md

```diff
@@ -37,6 +37,10 @@ are `systemd-devel`, `json-c-devel`, and `libcurl-devel` on RPM-based distros,
 and `libjson-c-dev`, `libsystemd-journal-dev`/`libsystemd-dev` and
 `libcurl-*-dev` on Debian-based distros.
 
+To build from Git you'll need `autoconf`, `automake` and `libtool` packages.
+On Debian you'll also need `pkg-config`. For creating RPM package `rpm-build`
```

Comment:
On Debian `pkg-config` is also needed even for a regular build, so we need to move it to the Debian's list in the paragraph above.

Also, please fix the commit subject as described in the previous review.

Thank you!